webapp/src/components/chat/shared/EditChatName.tsx

```diff
@@ -0,0 +1,111 @@
+import { useMsal } from '@azure/msal-react';
+import { Button } from '@fluentui/react-button';
+import { makeStyles, mergeClasses, shorthands, tokens } from '@fluentui/react-components';
+import { Input, InputOnChangeData } from '@fluentui/react-input';
+import { useEffect, useState } from 'react';
+import { AuthHelper } from '../../../libs/auth/AuthHelper';
+import { AlertType } from '../../../libs/models/AlertType';
+import { ChatService } from '../../../libs/services/ChatService';
+import { useAppDispatch, useAppSelector } from '../../../redux/app/hooks';
+import { RootState } from '../../../redux/app/store';
+import { addAlert } from '../../../redux/features/app/appSlice';
+import { editConversationTitle } from '../../../redux/features/conversations/conversationsSlice';
+import { Breakpoints } from '../../../styles';
+import { Checkmark20, Dismiss20 } from '../../shared/BundledIcons';
+
+const useClasses = makeStyles({
+    root: {
+        width: '100%',
+        ...Breakpoints.small({
+            display: 'none',
+        }),
+    },
+    buttons: {
+        display: 'flex',
+        alignSelf: 'end',
+    },
+    textButtons: {
+        ...shorthands.gap(tokens.spacingVerticalS),
+    },
+});
+
+interface IEditChatNameProps {
+    name: string;
+    chatId: string;
+    exitEdits: () => void;
+    textButtons?: boolean;
+}
+
+export const EditChatName: React.FC<IEditChatNameProps> = ({ name, chatId, exitEdits, textButtons }) => {
+    const classes = useClasses();
+    const dispatch = useAppDispatch();
+    const { instance, inProgress } = useMsal();
+    const chatService = new ChatService(process.env.REACT_APP_BACKEND_URI as string);
```

Comment:
Why not use useChat?

Reply:
This was copied from the implementation before -- added editChatName in useChat and calling that instead